.gitignore

```diff
@@ -1,2 +1,30 @@
-config.json
```

Comment:
SUPER SUPER SUPER SUPER important change here. 

Put your token in your `.env` file and delete this file on your machine. There are multiple ways of doing `.env` files like `.env.local` and `.env.development`, etc. But I like to keep it simple and just have a single `.env` file that is ignore in the `.gitignore` and then a single `.env.example` to give you an example of what it should look like. 

Doing it this way, you just run this command at the root of your project when you begin:

```cmd
cp .env.example .env
```

And then edit your `.env` with your actual tokens.

Reply:
definitely agree with this.  it makes things easier to test different environments.  One thing i would ask for, though, is that a note be added to a README.md about the `.env` file.

Reply:
Yeah I can do that.